yt_dlp/extractor/tele5.py

```diff
@@ -71,18 +184,64 @@ class Tele5IE(DPlayIE):  # XXX: Do not subclass from concrete IE
     }]
 
     def _real_extract(self, url):
-        video_id = self._match_id(url)
-        webpage = self._download_webpage(url, video_id)
-        player_element = self._search_regex(r'(<hyoga-player\b[^>]+?>)', webpage, 'video player')
-        player_info = extract_attributes(player_element)
-        asset_id, country, realm = (player_info[x] for x in ('assetid', 'locale', 'realm', ))
-        endpoint = compat_urlparse.urlparse(player_info['endpoint']).hostname
-        source_type = player_info.get('sourcetype')
-        if source_type:
-            endpoint = '%s-%s' % (source_type, endpoint)
-        try:
-            return self._get_disco_api_info(url, asset_id, endpoint, realm, country)
-        except ExtractorError as e:
-            if getattr(e, 'message', '') == 'Missing deviceId in context':
-                self.report_drm(video_id)
-            raise
+        content_regex = re.compile(
+            r'https?://(?:www\.)?(?P<environment>[^.]+)\.de/(?P<parent_slug>[^/]+)/(?P<slug>[^/?#&]+)')
+        m = content_regex.search(url)
```

Comment:
Read contributing guidelines for helper functions like `_search_regex`, `traverse_obj` etc